lib/cli.js

```diff
@@ -157,6 +157,21 @@ function runDevelopmentMode (testModulePath) {
     callback()
   })
 
+  vorpal.command('runto <stepNumber>')
+  .description('Run from current step to step number')
+  .action(function (args, callback) {
+    handleRun(co(function* () {
```

Comment:
There should be a space before `*`, per standard.

This project is using `standard` linter CLI directly, not through ESLint.

Sorry I forgot to make the CI run the linter.